tunnel/IpTunnel_admin.c

```diff
@@ -24,6 +24,9 @@
 #include "util/platform/Sockaddr.h"
 
 #include <stddef.h>
+#include <string.h>
+#include <inttypes.h>
+#include <stdlib.h>
```

Comment:
In general, pulling in headers from outside of cjdns should be avoided unless it is not possible to do so.